packages/beta/src/api/documents/feedbackApi.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 Cognite AS
+
+import {
+  BaseResourceAPI,
+  CursorAndAsyncIterator,
+  ItemsWrapper,
+} from '@cognite/sdk-core';
+
+import {
+  AggregateField,
+  DocumentFeedback,
+  DocumentFeedbackAggregateRequest,
+  DocumentFeedbackAggregateResponse,
+  DocumentFeedbackCreateItem,
+  DocumentFeedbackId,
+  FeedbackQueryParameters,
+  FeedbackStatus,
+} from '../../types';
+
+export class FeedbackAPI extends BaseResourceAPI<DocumentFeedback> {
+  public aggregates = (
+    fieldName: AggregateField
+  ): Promise<DocumentFeedbackAggregateResponse> => {
+    const content: DocumentFeedbackAggregateRequest = { field: fieldName };
+    return this.aggregateFeedbacks<DocumentFeedbackAggregateResponse>(content);
+  };
+
+  public create = (
+    feedbacks: DocumentFeedbackCreateItem[]
+  ): Promise<DocumentFeedback[]> => {
+    return this.createEndpoint<DocumentFeedbackCreateItem>(feedbacks);
+  };
+
+  public list = (
+    status: FeedbackStatus
+  ): CursorAndAsyncIterator<DocumentFeedback> => {
+    const parameter: FeedbackQueryParameters = { status: status };
+    return this.listEndpoint(this.callListEndpointWithGet, parameter);
+  };
+
+  public accept = (ids: number[]) => {
+    return this.postAcceptRejectFeedback(ids, 'accept');
+  };
+
+  public reject = (ids: number[]) => {
+    return this.postAcceptRejectFeedback(ids, 'reject');
+  };
+
+  private async postAcceptRejectFeedback(
+    ids: number[],
+    endpoint: acceptRejectEndpoint
+  ) {
+    const wrappedIds = ids.map(id => {
+      return <DocumentFeedbackId>{ id: id };
```

Comment:
We don't perform this kind of wrapping in other APIs.
Is there a specific reason for it here?

Reply:
It just felt a bit weird having to specify a list of objects that only has a single attribute. So this was only for convenience.

Reply:
Updated the code